src/reducers/CoursePlannerReducer.jsx

```diff
@@ -16,21 +16,247 @@ const initialState = {
       category: 'ME Math',
       courses: [
         {
-          name: 'Partial Differential Equations',
-          courseId: 'MTH3120',
+          courseCode: 'MTH3120',
+          title: 'Partial Differential Equations',
         },
         {
-          name: 'Numerical Methods and Scientific Computing',
-          courseId: 'MTH3150',
+          courseCode: 'MTH3150',
+          title: 'Numerical Methods and Scientific Computing',
         },
         {
-          name: 'Nonlinear Dynamics and Chaos',
-          courseId: 'MTH3170',
+          courseCode: 'MTH3170',
+          title: 'Nonlinear Dynamics and Chaos',
         },
       ],
       selected: ' ',
     },
   ],
+  allCourses: [
```

Comment:
IMO these should really be stored in a static file somewhere and read from that, so that there's a single source of truth for the application. (Maybe they should even be read from database, the first time through, so that they can be more easily managed by the registrar...)